terasoluna-gfw-parent/pom.xml

```diff
@@ -370,6 +370,26 @@
         <type>pom</type>
       </dependency>
       <!-- == End TERASOLUNA == -->
+
+      <!-- == Begin Spring Framework == -->
+      <dependency>
+        <groupId>org.springframework</groupId>
+        <artifactId>spring-framework-bom</artifactId>
+        <version>${springframework.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+      <!-- == End Spring Framework == -->
+
+      <!-- == Begin Spring Security == -->
+      <dependency>
+        <groupId>org.springframework.security</groupId>
+        <artifactId>spring-security-bom</artifactId>
+        <version>${springframework.security.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+      <!-- == End Spring Security == -->
```

Comment:
- [ ] I think that you should mention upgrading the library from the Spring IO Platform as a security vulnerability update.

Reply:
OK